### ISSUE TITLE
Extra comments in files

### ISSUE THREAD

**Issue by zoqaeski** [leozide/leocad#247]
Some of my files end up with millions of `0 !LEOCAD MODEL COMMENT` lines, causing them to balloon out to hundreds of MB in size, leading to excessive memory usage—I've had my computer crash with Leocad consuming 7.5 GB of memory. The only way to clean the files up seems to be editing them with a text editor and removing the unnecessary lines.

**Comment by leozide** [comment]
I've never seen that before, can you send me a file that has this problem or steps on how to get this to happen from a new model?

**Comment by zoqaeski** [comment]
I didn't notice it until inspecting the files with a text editor. Leocad kept eating up gigabytes of RAM and crashing the computer, and for some reason the files were huge. I've removed the extra lines from the files so they open properly now, but I've attached one of the models from a backup a few days old.

It's compressed with gzip to reduce it from 121 MB to ~400 kB.

[Z1a Sparrowhawk (Archangel redesign).ldr.gz](https://github.com/leozide/leocad/files/2240212/Z1a.Sparrowhawk.Archangel.redesign.ldr.gz)

As far as I know I didn't do anything special to cause the files to end up bloated, other than saving them after every design step.
